mountinfo/mountinfo_filters.go

```diff
@@ -18,7 +18,7 @@ type FilterFunc func(*Info) (skip, stop bool)
 // do not start with a specific prefix.
 func PrefixFilter(prefix string) FilterFunc {
 	return func(m *Info) (bool, bool) {
-		skip := !strings.HasPrefix(m.Mountpoint, prefix)
+		skip := m.Mountpoint != prefix && !strings.HasPrefix(m.Mountpoint, prefix+"/")
```

Comment:
I'm a bit in doubt if matching on a literal prefix was a valid use-case; if so, perhaps we need a new filter (e.g., `ParentFilter` or `ParentPathFilter`) ?

Reply:
We should also consider wether or not `prefix` itself should be included or not (i.e., if it's _inclusive_ of `prefix`).

If not, I need to remove `m.Mountpoint != prefix` from the above;

```suggestion
		skip := !strings.HasPrefix(m.Mountpoint, prefix+"/")
```

Reply:
For the second question (whether `PrefixFilter` should return the mountpoint if it equals to prefix) -- I believe it should (example, if I use `PrefixFilter("/usr")` and the `/usr` itself is a mount point, I'd expect it to be included.

Reply:
OK, I think the check here should be
```suggestion
skip := !strings.HasPrefix(m.Mountpoint+"/", prefix+"/")
```